lib/src/executable.dart

```diff
@@ -200,7 +204,7 @@ transformers:
 ///
 /// If [error] is passed, it's used in place of the usage message and the whole
 /// thing is printed to stderr instead of stdout.
-void _printUsage([String error]) {
+void _printUsage(String wordWrap(String message), [String error]) {
```

Comment:
I think it would be cleaner to just pass in the `Configuration` here.

Reply:
Done, I changed it to an optional named arg as well since there is one location where we don't have a configuration (if it fails to parse). Also changed error to a named instead of positional arg as a result of that.

Reply:
You can just use `Configuration.empty` when the args fail to get the default configuration.

Reply:
done